src/main/java/io/cdap/plugin/servicenow/sink/output/ServiceNowOutputFormat.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.servicenow.sink.output;
+
+import com.google.gson.JsonObject;
+import io.cdap.plugin.servicenow.sink.ServiceNowSinkConfig;
+import io.cdap.plugin.servicenow.sink.transform.ServiceNowRecordWriter;
+import io.cdap.plugin.servicenow.source.ServiceNowJobConfiguration;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.mapreduce.JobContext;
+import org.apache.hadoop.mapreduce.OutputCommitter;
+import org.apache.hadoop.mapreduce.OutputFormat;
+import org.apache.hadoop.mapreduce.RecordWriter;
+import org.apache.hadoop.mapreduce.TaskAttemptContext;
+
+import java.io.IOException;
+
+/**
+ *  ServiceNow Output Format Class
+ */
```

Comment:
Please add a valid javaDoc for this class.

Reply:
Please suggest a valid JavaDoc for this class.

Reply:
An OutputFormat that sends the output of a Hadoop job to the ServiceNow by determining the way these output key-value pairs are written in output files by RecordWriter. (Just a suggestion)

Reply:
Added 843ee483f38d0090b76f1a8be5e6bcf2a967bd1d